README.md

```diff
@@ -1097,7 +1097,7 @@ Other Style Guides
 
     // best
     import { es6 } from './AirbnbStyleGuide';
-    export default es6;
+    export es6;
```

Comment:
this is actually incorrect, as it will export a named export named "es6". We want `export default es6` so that there is only a default export.